store/src/java/com/zimbra/cs/account/AccountServiceException.java

```diff
@@ -417,4 +419,13 @@ public static AccountServiceException TWO_FACTOR_AUTH_REQUIRED() {
     public static AccountServiceException ALIAS_EXISTS(String name) {
         return new AccountServiceException("email address alias already exists: "+name, ALIAS_EXISTS, SENDERS_FAULT, null);
     }
+
+    public static AccountServiceException NEED_CAPTCHA() {
+        return new AccountServiceException("show CAPTCHA", NEED_CAPTCHA, SENDERS_FAULT, null);
+    }
+
+    public static AccountServiceException INVALID_CAPTCHA() {
+        return new AccountServiceException("CAPTCHA required", INVALID_CAPTCHA, SENDERS_FAULT, null);
```

Comment:
nit: for consistency with "show CAPTCHA", I (personally) would prefer "require CAPTCHA" over "CAPTCHA required"... your call though unless others chime in